src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -326,10 +392,14 @@
     <Error Condition="'$(TargetOSTrait)' == ''" Text="TargetOS [$(TargetOS)] is unknown so we don't know how to configure the test run for this project [$(MSBuildProjectName)]" />
 
     <PropertyGroup>
-      <RunnerScriptName Condition="'$(TargetOS)'=='Windows_NT'" >RunTests.cmd</RunnerScriptName>
-      <RunnerTemplateName Condition="'$(TargetOS)'=='Windows_NT'" >RunnerTemplate.Windows.txt</RunnerTemplateName>
-      <RunnerScriptName Condition="'$(TargetOS)'!='Windows_NT'" >RunTests.sh</RunnerScriptName>
-      <RunnerTemplateName Condition="'$(TargetOS)'!='Windows_NT'" >RunnerTemplate.Unix.txt</RunnerTemplateName>
+      <RunnerScriptName Condition="'$(TargetOS)' == 'Windows_NT'">RunTests.cmd</RunnerScriptName>
+      <RunnerScriptName Condition="'$(TargetOS)' != 'Windows_NT'">RunTests.sh</RunnerScriptName>
+
+      <!-- In UAP we don't want the repro steps printed -->
+      <RunnerTemplateName Condition="'$(TargetOS)' == 'Windows_NT' AND '$(BuildingUAPVertical)' != 'true'">RunnerTemplate.Windows.txt</RunnerTemplateName>
+      <RunnerTemplateName Condition="'$(TargetOS)' == 'Windows_NT' AND ('$(BuildingUAPVertical)' == 'true' OR '$(UseDotNetNativeToolchain)' == 'true')">RunnerTemplate.Quiet.Windows.txt</RunnerTemplateName>
+      <RunnerTemplateName Condition="'$(TargetOS)' != 'Windows_NT' AND '$(BuildingUAPVertical)' != 'true'">RunnerTemplate.Unix.txt</RunnerTemplateName>
+      <RunnerTemplateName Condition="'$(TargetOS)' != 'Windows_NT' AND ('$(BuildingUAPVertical)' == 'true' OR '$(UseDotNetNativeToolchain)' == 'true')">RunnerTemplate.Quiet.Unix.txt</RunnerTemplateName>
```

Comment:
I don't think UAPVertical will be true when in Unix.

Reply:
right, there is no BuildingUAPVertical or .NET Native Toolchain on Unix right? Then I can just the Unix line.

Reply:
or I'll just keep it for in case?

Reply:
UseDotNetNativeToolchain I think will be enabled for netcoreapp https://github.com/dotnet/buildtools/pull/2095

Reply:
ok then I'll just leave it as it is. Not worth changing that. Maybe some day there will be targeting pack for Unix for UWP :D